browserslist

```diff
@@ -0,0 +1,3 @@
+last 2 Chrome versions
+last 2 Firefox versions
+IE 11
```

Comment:
Hmm this [`browserslist`](https://github.com/browserslist/browserslist) is so simple it looks curious